catalog/spatial/registry/registry-app/src/main/resources/features.xml

```diff
@@ -47,10 +47,11 @@
         <bundle>mvn:org.codice.ddf.registry/registry-admin-local-ui/${project.version}</bundle>
         <bundle>mvn:org.codice.ddf.registry/registry-report-action-provider/${project.version}</bundle>
         <bundle>mvn:org.codice.ddf.registry/registry-source-configuration-handler/${project.version}</bundle>
+        <bundle>mvn:org.codice.ddf.registry/registry-publication-update-handler/${project.version}</bundle>
     </feature>
 
     <feature name="registry-app" install="auto" version="${project.version}"
-             description="DDF Registry contains the base registry components, plugins, sources and interfaces needed for the registry to function">
+             description="DDF Registry contains the base registry components, plugins, sources and interfaces needed for the registry to function.::DDF Registry">
```

Comment:
Yes it does! Found out thats how you set the app name so now it will show up as DDF Registry instead of Registry APP which Michael didn't like.